packages/server/api/.env

```diff
@@ -15,4 +15,4 @@ AP_TELEMETRY_ENABLED=false
 AP_TRIGGER_DEFAULT_POLL_INTERVAL=1
 AP_WEBHOOK_TIMEOUT_SECONDS=30
 AP_PIECES_SYNC_MODE=NONE
-AP_WEBHOOK_URL="http://localhost:3000"
\ No newline at end of file
+AP_WEBHOOK_URL="https://legendary-garbanzo-q9995r99p4pfx6px-3000.app.github.dev"
```

Comment:
revert file changes?

Reply:
Yes, What I need to do, could not understand,

I followed the tutorial of pieces development. 

Let me know, what I need to change in them, you can edit from your side, if possible

Reply:
If I'm not mistaken, this file should not be modified when adding a piece. Just remove your changes.